config/v1/types_infrastructure.go

```diff
@@ -308,7 +308,7 @@ type PlatformSpec struct {
 	// +optional
 	Nutanix *NutanixPlatformSpec `json:"nutanix,omitempty"`
 
-	// ExternalPlatformType represents generic infrastructure provider.
+	// External contains settings specific to the External platform.
```

Comment:
Should be this, if you can update and then run `make update` that would be appreciated
```suggestion
	// external contains settings specific to the External platform.
```

Reply:
@JoelSpeed Sure but why it should be `external` and `External`? Other comments start with the uppercase letter, no?

Reply:
Our conventions for APIs dictate that you should use the `json` tag version of the name at the beginning of the comment, so that, when someone does an `oc explain` on the object, the comment starts with the name that they are familiar with, as this is the serialized version that they use and see.

The existing fields here are not compliant, but that doesn't mean we should make the same mistake